docs/release_notes/v0.10.0.md

```diff
@@ -0,0 +1,457 @@
+  
+# Dapr 0.10.0
+
+We're happy to announce the release of Dapr 0.10.0!
+
+As Dapr is approaching a 1.0 release, this release focuses on operational concerns in production environments, stability improvements, test infrastructure enhancements and as always, community requests and user feedback in the form of new features and bug fixes.
+
+This release contains all the *expected* breaking changes for the 1.0 release.
+
+**Highlights**
+
+* Support for multiple Pub/Sub components ([Publisher](https://github.com/dapr/docs/tree/master/howto/publish-topic#publish-a-topic), [Subscriber](https://github.com/dapr/docs/tree/master/howto/consume-topic#subscribe-to-topics))
+* [Support for Windows Containers]((https://github.com/dapr/docs/tree/master/howto/kubernetes-windows)
+* [New Bulk Get API for State](https://github.com/dapr/docs/blob/master/reference/api/state_api.md#get-bulk-state)
+* [New Transactions API for State](https://github.com/dapr/docs/blob/master/reference/api/state_api.md#state-transactions)
+* Highly available control plane on Kubernetes
+* [Production deployment and upgrade guidelines for Kubernetes](https://github.com/dapr/docs/tree/master/howto/deploy-k8s-prod)
+* Improved [Go](https://github.com/dapr/go-sdk) and [Python](https://github.com/dapr/python-sdk) SDKs
+* [Improved Dashboard with new features](https://github.com/dapr/dashboard)
+* New CLI commands (view root cert expiry, export TLS certs, init on k8s with a custom namespace)
+* New components: [InfluxDB binding](https://github.com/dapr/docs/blob/master/reference/specs/bindings/influxdb.md), [Non-interactive OAuth2 middleware](https://github.com/dapr/docs/tree/master/howto/authorization-with-oauth#define-a-client-credentials-grant-component), [Azure Blob Storage state store](https://github.com/dapr/docs/blob/master/howto/setup-state-store/setup-azure-blobstorage.md)
+
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
+
+If you're new to Dapr, visit the [getting started](https://github.com/dapr/docs/tree/master/getting-started) page and familiarize yourself with Dapr.
```

Comment:
The 1st sentence refers to contributor. The second one, to the user. Maybe move the contributor focused one above the Highlights section?

Reply:
Changed